src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -907,6 +930,9 @@ private bool IsAttributeDeclarationTerminator()
                 || this.IsPossibleAttributeDeclaration(); // start of a new one...
         }
 
+        private bool IsAttributeTarget()
```

Comment:
>IsAttributeTarget [](http://example.com/codeflow?start=21&length=17)

I find the name for this helper extremely confusing/misrepresenting. The method doesn't even check if we are right after a`[` token. Consider using the name that accurately reflects what the method checks for. For example, "IsSomeWordFollowedByColon".

Reply:
So, this follows the naming we have all over the parser.  The `Is` methods are very often for asking if we're on a particular construct, given that teh caller sensibly can ask that sort of question.  Given that both callers are effectively the attribute checking/parsing code, asking if hte position we're at is an attribute target is sensible (and fits the rest of the parser).